src/scene/light.js

```diff
@@ -326,15 +332,15 @@ class Light {
     uploadAreaLightLUTs() {
 
         function createTexture(device, data, format) {
-            var tex = new pc.Texture(device, {
+            var tex = new Texture(device, {
                 width: 64,
                 height: 64,
                 format: format,
-                addressU: pc.ADDRESS_CLAMP_TO_EDGE,
-                addressV: pc.ADDRESS_CLAMP_TO_EDGE,
-                type: pc.TEXTURETYPE_RGBE,
-                magFilter: pc.FILTER_LINEAR,
-                minFilter: pc.FILTER_NEAREST,
+                addressU: ADDRESS_CLAMP_TO_EDGE,
+                addressV: ADDRESS_CLAMP_TO_EDGE,
+                type: TEXTURETYPE_DEFAULT,
```

Comment:
This is changing more than removing the `pc.`. What's the justification here?

Reply:
A bug I introduced by copy & paste when writing this code.

Reply:
OK, so it's correct now? Cool.

Reply:
yes it's just normal RGBA texture without a special meaning for the alpha.